### ISSUE TITLE
Authentication error message when using --deploy -- deploy-hook after 2023 Sep 25 update

### ISSUE THREAD

**Issue by Eridani78** [acmesh-official/acme.sh#4814]
After update, I get the following message when launching the deploy function :

[Fri Sep 29 03:05:02 UTC 2023] Logging into 172.17.0.1:5000
[Fri Sep 29 03:05:02 UTC 2023] Unable to authenticate to http://172.17.0.1:5000/ - check your username & password.
[Fri Sep 29 03:05:02 UTC 2023] If two-factor authentication is enabled for the user, set SYNO_Device_ID.
[Fri Sep 29 03:05:02 UTC 2023] Error deploy for domain:ebohome.net
[Fri Sep 29 03:05:02 UTC 2023] Deploy error.

It seems that the words entry.cgi disappeared from the url (see part of the log below).

Steps to reproduce
------------------

Debug log (acme.sh.log)
-----------------

```
Before 2023 Sep 25 update
url='http://172.17.0.1:5000/webapi/entry.cgi?api=SYNO.API.Auth&version=7&method=login&format=sid&account= .....

After 2023 Sep 25 update
url='http://172.17.0.1:5000/webapi/             ?api=SYNO.API.Auth&version=7&method=login&format=sid&account= .....

```

**Comment by nicobo01** [comment]
Hello,

It's similar : #4765

**Comment by winromulus** [comment]
Fixed in #4809

**Comment by Eridani78** [comment]
@winromulus ,
Many thanks.
I spent hours trying to figure this out !
Hopefully, you fixed it. Waiting for the update now ...
